### PR TITLE
LPS-161075 KB EmptyState for templates and suggestion's sidebar

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Templates: 
Screenshot 2022-08-22 at 12 31 21

Suggestions:
Screenshot 2022-08-22 at 12 28 41

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/components/TemplatesPanel.js [hunk 1]
```diff
@@ -12,10 +12,23 @@
...
+import ClayEmptyState from '@clayui/empty-state';
+import PropTypes from 'prop-types';
 import React from 'react';
 
-export default function TemplatesPanel() {
-    return (
-        <h1>Templates Panel</h1>
-    );
+export default function TemplatesPanel({items}) {
+	return items ? (
```
Comment (boton) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/components/TemplatesPanel.js:20]: For an empty array, we will need the ClayEmptyStat too. 
I think it's better to check the length too `items?.length`

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/components/TemplatesPanel.js [hunk 2]
```diff
@@ -12,10 +12,23 @@
...
+		<h1>Templates Panel</h1>
```
Comment (boton) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/components/TemplatesPanel.js:21]: Can we return a null for now and avoid hardcoded text?

**Review by boton — CHANGES_REQUESTED** [review]

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/components/TemplatesPanel.js (hunk 1 above)
Comment (ambrinchaudhary) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/components/TemplatesPanel.js]: done!

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/components/TemplatesPanel.js (hunk 2 above)
Comment (ambrinchaudhary) [reply modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/components/TemplatesPanel.js]: done!

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:stop

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by ambrinchaudhary** [comment]
ci:forward

**Pull request closed without merging**